channels/us.m3u

```diff
@@ -349,49 +317,25 @@ http://n1.klowdtv.net/live1/awe_720p/chunks.m3u8
 https://aweencore-vizio.amagi.tv/playlist.m3u8
 #EXTINF:-1 tvg-id="AWEEncore.us" tvg-name="AWE Encore" tvg-country="US" tvg-language="" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/60482f63cfbf071d6151ca95" group-title="News",AWE Encore
 https://cdn.herringnetwork.com/80A4DFF/awee_nva/AWE_Encore.m3u8
-#EXTINF:-1 tvg-id="AXN.us" tvg-name="AXN" tvg-country="US" tvg-language="Spanish" tvg-logo="" group-title="Movies",AXN
-http://136.243.177.164/AXN/playlist.m3u8
-#EXTINF:-1 tvg-id="AXNIndonesia.us" tvg-name="AXN [Indonesia]" tvg-country="US" tvg-language="" tvg-logo="" group-title="",AXN [Indonesia]
-http://210.210.155.35/session/37ee409e-73b5-11e9-b657-a0899505893d/uq2663/h/h141/01.m3u8
```

Comment:
I bet it has become geo-blocked, returns a legit m3u8 that doesn't play

Reply:
It closes off when I ask to connect on my side.